eclair-core/src/main/scala/fr/acinq/eclair/blockchain/electrum/ElectrumClient.scala

```diff
@@ -590,6 +593,14 @@ object ElectrumClient {
             UnspentItem(ByteVector32.fromValidHex(tx_hash), tx_pos, value, height)
           })
           ScriptHashListUnspentResponse(scripthash, items)
+        case GetTransactionIdFromPosition(_, _, false) =>
+          val JString(txid) = json.result
+          GetTransactionIdFromPositionResponse(ByteVector32.fromValidHex(txid), Nil)
+        case GetTransactionIdFromPosition(_, _, true) =>
+          val JString(txid) = json.result \ "tx_hash"
```

Comment:
nit: for consistency with other methods

```suggestion
          val JString(tx_hash) = json.result \ "tx_hash"
```

Reply:
Yeah it was a bit hard to choose, Electrum uses both tx_hash and tx_id and we also mix the two :)